Formula/amazon-efs-utils.rb

```diff
@@ -216,6 +218,10 @@ def caveats
         sudo rm /Library/Filesystems/efs.fs/Contents/Resources/mount_efs
 
       To enable watchdog for TLS mounts:
+        * Prerequisite for mac2.metal *
+          Mac2 has different installation path /opt/homebrew comparing with Mac1, run following commands to update the paths in watchdog.plist to use this
+          sed s@/usr/local/@/opt/homebrew/@g /opt/homebrew/Cellar/amazon-efs-utils/${VERSION}/libexec/amazon-efs-mount-watchdog.plist
```

Comment:
Question: can we do this in the formula during installation and eliminate the need for intervention altogether?

Reply:
Thanks for the comment. Please check updated PR out